eva_sub_cli_processing/process_jobs.py

```diff
@@ -0,0 +1,99 @@
+from datetime import datetime
+
+import requests
+from ebi_eva_common_pyutils.common_utils import pretty_print
+from ebi_eva_common_pyutils.config import cfg
+from ebi_eva_common_pyutils.logger import AppLogger
+from retry import retry
+
+
+def _url_build(*args, **kwargs):
+    url = cfg.query('submissions', 'webservice', 'url') + '/' + '/'.join(args)
+    if kwargs:
+        return url + '?' + '&'.join(f'{k}={v}' for k, v in kwargs.items())
+    else:
+        return url
+
+
+@retry(tries=5, backoff=2, jitter=.5)
+def _get_submission_api(url):
+    auth = (cfg.query('submissions', 'webservice', 'admin_username'), cfg.query('submissions', 'webservice', 'admin_password'))
+    response = requests.get(url, auth=auth)
+    response.raise_for_status()
+    return response.json()
+
+
+@retry(tries=5, backoff=2, jitter=.5)
+def _put_submission_api(url):
+    auth = (cfg.query('submissions', 'webservice', 'admin_username'), cfg.query('submissions', 'webservice', 'admin_password'))
+    response = requests.put(url, auth=auth)
+    response.raise_for_status()
+    return response.json()
+
+
+def process_submissions():
+
+    scanner = SubmissionScanner()
+    ready_submissions = scanner.scan()
+
+    if not ready_submissions:
+        return 0
+    else:
+        # Only process the first new dataset found. Run through Cron, this will result in one new pipeline
+        # being kicked off per interval.
+        for ready_submission in ready_submissions:
+            return _process_submission(ready_submission)
+
+
+def _process_submission(submission):
+    # Create submission processing directory
```

Comment:
```suggestion
    # TODO: Create submission processing directory
```
I'm not sure this would happen here or inside `submission.start()`, but either way I'm assuming it's a TODO.